components/FrontPage/Section/Wrapper.vue

```diff
@@ -0,0 +1,35 @@
+<script lang="ts" setup>
+const props = defineProps<{
+  tag: Tag
+}>()
+</script>
+
+<template>
+  <div class="frontpage-section-wrapper mt-6">
+    <h2 class="section-title title is-2 mx-6">
+      <NuxtLink :to="'/tag/' + tag"
+        >{{ tag }} <span class="more">&#x2192; see all</span></NuxtLink
+      >
+    </h2>
+    <hr class="light" />
+    <section class="section topic pt-3 pb-6">
+      <slot></slot>
+    </section>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+// Styles for the Sections on the home page
+h2.section-title.title.is-2 a {
+  padding-top: 8rem !important;
+  color: $black !important; 
+  &:hover {
+    color: $link !important;
```

Comment:
I think we can remove the `!important` properties here. These CSS rules seem to work just as well without them.

Reply:
Fixed!